Packet++/header/Layer.h

```diff
@@ -124,12 +124,12 @@ namespace pcpp
 		 * Copy the raw data of this layer to another array
 		 * @param[out] toArr The destination byte array
 		 */
-		void copyData(uint8_t* toArr);
+		void copyData(uint8_t* toArr) const;
 
 
 		// implement abstract methods
 
-		uint8_t* getDataPtr(size_t offset = 0) { return (uint8_t*)(m_Data + offset); }
+		uint8_t* getDataPtr(size_t offset = 0) const { return m_Data + offset; }
```

Comment:
this might cause compilation warning on some platforms

Reply:
The changes have been undone